ui/app/templates/vault/cluster/access/oidc/assignments/index.hbs

```diff
@@ -0,0 +1,64 @@
+<Toolbar>
+  <ToolbarActions>
+    <ToolbarLink @type="add" @params={{array "vault.cluster.access.oidc.assignments.create"}}>
+      Create assignment
+    </ToolbarLink>
+  </ToolbarActions>
+</Toolbar>
+
+{{#each this.model as |model|}}
+  <LinkedBlock
+    class={{concat "list-item-row" (if (eq model.name "allow_all") " is-disabled")}}
```

Comment:
Just an FYI about handlebar file syntax (that I learned recently!) this line can also be written without using the `concat` helper:
 
`class="list-item-row {{if (eq model.name 'allow_all') 'is-disabled'}}"`